src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -101,78 +100,59 @@ const OscapContent = () => {
+    },
+    {
+      skip: complianceType === 'openscap',
+    }
+  );
 
   return (
     <Form>
       <Title headingLevel="h1" size="xl">
         {complianceEnabled ? 'Compliance' : 'OpenSCAP profile'}
       </Title>
+      <Text>
+        Below you can select which Insights compliance policy or OpenSCAP
+        profile your image will be compliant to. Insights compliance allows the
+        use of tailored policies, whereas OpenSCAP gives you the default
+        versions. This will automatically help monitor the adherence of your
+        registered RHEL systems to a selected policy or profile.
+      </Text>
       {complianceEnabled && (
-        <FormGroup>
-          <Radio
-            id="openscap radio openscap type"
-            label="OpenSCAP"
-            name="oscap-radio-openscap"
-            isChecked={complianceType === 'openscap'}
-            onChange={() => handleTypeChange('openscap')}
-          />
-          <Radio
-            id="openscap radio compliance type"
-            label="Insights compliance"
-            name="oscap-radio-compliance"
-            isChecked={complianceType === 'compliance'}
+        <ToggleGroup aria-label="Default with single selectable">
+          <ToggleGroupItem
+            text="Compliance policies"
+            buttonId="toggle-group-compliance"
+            isSelected={complianceType === 'compliance'}
             onChange={() => handleTypeChange('compliance')}
           />
-        </FormGroup>
-      )}
-      {(!complianceEnabled || complianceType === 'openscap') && (
-        <Text>
-          OpenSCAP enables you to automatically monitor the adherence of your
-          registered RHEL systems to a selected regulatory compliance profile.
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_AND_VULN_SCANNING_URL}
-          >
-            Documentation
-          </Button>
-        </Text>
-      )}
-      {complianceType === 'compliance' && (
-        <Text>
-          Insights compliance enables you to monitor the adherence of your
-          registered RHEL systems to a selected compliance policy.
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_URL}
-          >
-            Define new policies in Insights Compliance
-          </Button>
-          <br />
-          <Button
-            component="a"
-            target="_blank"
-            variant="link"
-            icon={<ExternalLinkAltIcon />}
-            iconPosition="right"
-            isInline
-            href={COMPLIANCE_AND_VULN_SCANNING_URL}
-          >
-            Documentation
-          </Button>
```

Comment:
We're losing the compliance and documentation links in the process. I got that they're not in the mock then, but still wonder if we shouldn't keep them or give the users another option to quickly navigate to compliance (there's a link in the alert, but that renders only when there are no policies defined if I understand correctly)

@lucasgarfield any ideas?